docs/docs/configuration/index.md

```diff
@@ -37,6 +37,8 @@ It is not recommended to copy this full configuration file. Only specify values
 
 :::
 
+**Note:**  Dynamic configuration values are currently supported in the following sections: `mqtt.password` and `cameras.<name>.ffmpeg.inputs.[].path`.  Dynamic values can be set using environment variables with names starting with `FRIGATE_`.  For example, `{FRIGATE_MQTT_PASSWORD}`, `{FRIGATE_RTSP_USER}`, `{FRIGATE_RTSP_PASSWORD}`.
```

Comment:
Definitely a good idea to call out, but I don't think there is a reason to be as technical. Can just say it is only supported for MQTT Password and ffmpeg paths